src/content/docs/en/guides/testing.mdx

```diff
@@ -16,6 +16,106 @@ A Vite-native unit test framework with ESM, TypeScript and JSX support powered b
 
 See the [Astro + Vitest starter template](https://github.com/withastro/astro/tree/latest/examples/with-vitest) on GitHub.
 
+## Cypress
+
+Cypress is a front-end testing tool built for the modern web. Cypress enables you to write end-to-end tests for your Astro site.
+
+### Installation
+
+You can install Cypress using the package manager of your choice.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install -D cypress
+  ```
+
+  This will install Cypress locally as a dev dependency for your project.
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add cypress --save-dev
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add cypress --dev
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuration
+
+In the root of your project, create a `cypress.config.js` file with the following content:
+
+```js title="cypress.config.js"
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  e2e: {
+    supportFile: false
+  }
+})
+```
+
+### Create your first Cypress test
+
+1. Choose a page to test. We will use the example `index.astro` page below.
```

Comment:
```suggestion
1. Choose a page to test. This example will test the example page `index.astro` below.
```
We don't use "we" in our instructions as per our style guide. When necessary, "you" is OK. Otherwise, we rewrite to avoid addressing any particular person and try to keep things as neutral as possible.

(And yes, I know the example below uses "we".. .and that should also be corrected. So, can't fault you for following what you see in front of you on the same page! Baby improvement steps! Let's at least catch *this* one while we're here... 😅 )